public/index.html

```diff
@@ -250,6 +246,8 @@ <h1>Order Summary</h1>
   <script src="https://cdn.rawgit.com/davidshimjs/qrcodejs/gh-pages/qrcode.min.js"></script>
   <!-- App -->
   <script src="/javascripts/store.js"></script>
+  <script src="https://storage.googleapis.com/stripe-tracer-js/aquarium-0.0.3.js"></script>
```

Comment:
Adding the tracer client to the frontend -- we could also probably do this at run-time within the Tracer specific code to avoid loading this for normal scenarios. Seems kinda lurky but let me know if you want me to and I can add that.